AsyncDisplayKit/ASCellNode.m

```diff
@@ -123,7 +123,6 @@ - (void)setText:(NSString *)text
   _textNode.attributedString = [[NSAttributedString alloc] initWithString:_text
                                                                attributes:@{NSFontAttributeName: [UIFont systemFontOfSize:kFontSize]}];
 
-  [self invalidateCalculatedLayout];
```

Comment:
Added animation for Kitten node's internal relayout (image and text nodes swapped) in 210a89e.